[Flaky test] TestDropPacketDedupLog

**Describe the bug**
I observed the `TestDropPacketDedupLog` unit test fail on Windows (`Unit test (windows-2019)` CI job)

```text
2022-04-12T18:46:51.6946938Z --- FAIL: TestDropPacketDedupLog (0.27s)
2022-04-12T18:46:51.6948103Z     audit_logging_test.go:174: 
2022-04-12T18:46:51.6948759Z         	Error Trace:	audit_logging_test.go:174
2022-04-12T18:46:51.6950135Z         	Error:      	"2022/04/12 AntreaPolicyIngressRule AntreaNetworkPolicy:default/test Drop 0 0.0.0.0 35402 1.1.1.1 80 TCP 60\n" does not contain "AntreaPolicyIngressRule AntreaNetworkPolicy:default/test Drop 0 0.0.0.0 35402 1.1.1.1 80 TCP 60 [2"
2022-04-12T18:46:51.6951207Z         	Test:       	TestDropPacketDedupLog
2022-04-12T18:46:51.6951667Z I0412 18:46:35.134745    5020 channel.go:83] "Starting SubscribableChannel" name="PodUpdate"
2022-04-12T18:46:51.6952229Z I0412 18:46:35.304770    5020 channel.go:87] "Stopping SubscribableChannel" name="PodUpdate"
2022-04-12T18:46:51.6952774Z I0412 18:46:35.304770    5020 channel.go:83] "Starting SubscribableChannel" name="PodUpdate"
2022-04-12T18:46:51.6953286Z I0412 18:46:35.327522    5020 channel.go:87] "Stopping SubscribableChannel" name="PodUpdate"
2022-04-12T18:46:51.6955397Z I0412 18:46:35.328028    5020 channel.go:83] "Starting SubscribableChannel" name="PodUpdate"
2022-04-12T18:46:51.6955985Z I0412 18:46:35.561984    5020 channel.go:87] "Stopping SubscribableChannel" name="PodUpdate"
2022-04-12T18:46:51.6956507Z I0412 18:46:35.562984    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.6956995Z I0412 18:46:35.563983    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.6957503Z I0412 18:46:35.564984    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.6958116Z I0412 18:46:35.565984    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.6958631Z I0412 18:46:35.566984    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.6959338Z I0412 18:46:35.567985    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.6960287Z I0412 18:46:35.568984    5020 fqdn.go:181] "Unable to derive DNS server from the kube-dns Service, will fall back to local resolver"
2022-04-12T18:46:51.6961455Z I0412 18:46:35.778262    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.6962082Z I0412 18:46:35.779262    5020 networkpolicy_controller.go:429] Waiting for Antrea client to be ready
2022-04-12T18:46:51.6962631Z I0412 18:46:35.779262    5020 networkpolicy_controller.go:440] Antrea client is ready
2022-04-12T18:46:51.6963211Z I0412 18:46:35.780262    5020 networkpolicy_controller.go:455] Waiting for all watchers to complete full sync
2022-04-12T18:46:51.6963790Z I0412 18:46:35.780262    5020 networkpolicy_controller.go:646] Starting watch for AddressGroup
2022-04-12T18:46:51.6964524Z I0412 18:46:35.780262    5020 networkpolicy_controller.go:659] Started watch for AddressGroup
2022-04-12T18:46:51.6965268Z I0412 18:46:35.780262    5020 networkpolicy_controller.go:646] Starting watch for AppliedToGroup
2022-04-12T18:46:51.6965988Z I0412 18:46:35.780262    5020 networkpolicy_controller.go:659] Started watch for AppliedToGroup
2022-04-12T18:46:51.6966517Z I0412 18:46:35.780262    5020 networkpolicy_controller.go:646] Starting watch for NetworkPolicy
2022-04-12T18:46:51.6967071Z I0412 18:46:35.780262    5020 networkpolicy_controller.go:659] Started watch for NetworkPolicy
2022-04-12T18:46:51.6967651Z I0412 18:46:35.780262    5020 networkpolicy_controller.go:689] Received 1 init events for NetworkPolicy
2022-04-12T18:46:51.6968546Z I0412 18:46:35.780262    5020 networkpolicy_controller.go:244] NetworkPolicy K8sNetworkPolicy:ns1/policy1 applied to Pods on this Node
2022-04-12T18:46:51.6969373Z I0412 18:46:35.954691    5020 networkpolicy_controller.go:689] Received 1 init events for AddressGroup
2022-04-12T18:46:51.6970134Z I0412 18:46:36.068903    5020 networkpolicy_controller.go:689] Received 1 init events for AppliedToGroup
2022-04-12T18:46:51.6970936Z I0412 18:46:36.068903    5020 networkpolicy_controller.go:457] All watchers have completed full sync, installing flows for init events
2022-04-12T18:46:51.6971535Z I0412 18:46:36.069414    5020 networkpolicy_controller.go:461] Starting NetworkPolicy workers now
2022-04-12T18:46:51.6972336Z I0412 18:46:36.069414    5020 networkpolicy_controller.go:467] Starting IDAllocator worker to maintain the async rule cache
2022-04-12T18:46:51.6972940Z I0412 18:46:36.069414    5020 status_controller.go:209] Starting NetworkPolicy StatusController
2022-04-12T18:46:51.6973745Z I0412 18:46:36.070425    5020 status_controller.go:216] Shutting down NetworkPolicy StatusController
2022-04-12T18:46:51.6974471Z I0412 18:46:36.071428    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.6975229Z I0412 18:46:36.071428    5020 networkpolicy_controller.go:429] Waiting for Antrea client to be ready
2022-04-12T18:46:51.6975946Z I0412 18:46:36.071428    5020 networkpolicy_controller.go:440] Antrea client is ready
2022-04-12T18:46:51.6977052Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:455] Waiting for all watchers to complete full sync
2022-04-12T18:46:51.6977668Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:646] Starting watch for AppliedToGroup
2022-04-12T18:46:51.6978203Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:659] Started watch for AppliedToGroup
2022-04-12T18:46:51.6978763Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:646] Starting watch for AddressGroup
2022-04-12T18:46:51.6979319Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:659] Started watch for AddressGroup
2022-04-12T18:46:51.6979875Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:646] Starting watch for NetworkPolicy
2022-04-12T18:46:51.6980430Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:659] Started watch for NetworkPolicy
2022-04-12T18:46:51.6981004Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:689] Received 1 init events for AddressGroup
2022-04-12T18:46:51.6981553Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:689] Received 1 init events for AppliedToGroup
2022-04-12T18:46:51.6982151Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:689] Received 1 init events for NetworkPolicy
2022-04-12T18:46:51.6982818Z I0412 18:46:36.072428    5020 networkpolicy_controller.go:244] NetworkPolicy K8sNetworkPolicy:ns1/policy1 applied to Pods on this Node
2022-04-12T18:46:51.6983546Z I0412 18:46:36.073429    5020 networkpolicy_controller.go:457] All watchers have completed full sync, installing flows for init events
2022-04-12T18:46:51.6984190Z E0412 18:46:36.073429    5020 networkpolicy_controller.go:580] Rule b79e1338e32f78b2 is effective but not realizable
2022-04-12T18:46:51.6984782Z I0412 18:46:36.073429    5020 networkpolicy_controller.go:461] Starting NetworkPolicy workers now
2022-04-12T18:46:51.6985410Z I0412 18:46:36.073429    5020 networkpolicy_controller.go:467] Starting IDAllocator worker to maintain the async rule cache
2022-04-12T18:46:51.6986122Z I0412 18:46:36.073429    5020 status_controller.go:209] Starting NetworkPolicy StatusController
2022-04-12T18:46:51.6986835Z I0412 18:46:36.173722    5020 status_controller.go:216] Shutting down NetworkPolicy StatusController
2022-04-12T18:46:51.6987374Z I0412 18:46:36.174718    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.6987898Z I0412 18:46:36.175717    5020 networkpolicy_controller.go:429] Waiting for Antrea client to be ready
2022-04-12T18:46:51.6988396Z I0412 18:46:36.175717    5020 networkpolicy_controller.go:440] Antrea client is ready
2022-04-12T18:46:51.6988928Z I0412 18:46:36.175717    5020 networkpolicy_controller.go:455] Waiting for all watchers to complete full sync
2022-04-12T18:46:51.6989474Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:646] Starting watch for AppliedToGroup
2022-04-12T18:46:51.6990007Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:659] Started watch for AppliedToGroup
2022-04-12T18:46:51.6991220Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:646] Starting watch for NetworkPolicy
2022-04-12T18:46:51.6992078Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:659] Started watch for NetworkPolicy
2022-04-12T18:46:51.6992857Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:646] Starting watch for AddressGroup
2022-04-12T18:46:51.6993418Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:659] Started watch for AddressGroup
2022-04-12T18:46:51.6994024Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:689] Received 1 init events for AppliedToGroup
2022-04-12T18:46:51.6994621Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:689] Received 1 init events for NetworkPolicy
2022-04-12T18:46:51.6995309Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:244] NetworkPolicy K8sNetworkPolicy:ns1/policy1 applied to Pods on this Node
2022-04-12T18:46:51.6996368Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:689] Received 1 init events for AddressGroup
2022-04-12T18:46:51.6997035Z I0412 18:46:36.176718    5020 networkpolicy_controller.go:457] All watchers have completed full sync, installing flows for init events
2022-04-12T18:46:51.6997687Z I0412 18:46:36.177718    5020 networkpolicy_controller.go:461] Starting NetworkPolicy workers now
2022-04-12T18:46:51.6998318Z I0412 18:46:36.177718    5020 networkpolicy_controller.go:467] Starting IDAllocator worker to maintain the async rule cache
2022-04-12T18:46:51.6998921Z I0412 18:46:36.177718    5020 status_controller.go:209] Starting NetworkPolicy StatusController
2022-04-12T18:46:51.6999587Z I0412 18:46:36.178718    5020 networkpolicy_controller.go:228] NetworkPolicy K8sNetworkPolicy:ns1/policy1 no longer applied to Pods on this Node
2022-04-12T18:46:51.7000254Z I0412 18:46:36.178718    5020 status_controller.go:216] Shutting down NetworkPolicy StatusController
2022-04-12T18:46:51.7000842Z I0412 18:46:36.179718    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7001364Z I0412 18:46:36.179718    5020 networkpolicy_controller.go:429] Waiting for Antrea client to be ready
2022-04-12T18:46:51.7001910Z I0412 18:46:36.179718    5020 networkpolicy_controller.go:440] Antrea client is ready
2022-04-12T18:46:51.7002508Z I0412 18:46:36.180718    5020 networkpolicy_controller.go:455] Waiting for all watchers to complete full sync
2022-04-12T18:46:51.7003092Z I0412 18:46:36.180718    5020 networkpolicy_controller.go:646] Starting watch for AddressGroup
2022-04-12T18:46:51.7003640Z I0412 18:46:36.180718    5020 networkpolicy_controller.go:659] Started watch for AddressGroup
2022-04-12T18:46:51.7004205Z I0412 18:46:36.180718    5020 networkpolicy_controller.go:646] Starting watch for AppliedToGroup
2022-04-12T18:46:51.7004777Z I0412 18:46:36.180718    5020 networkpolicy_controller.go:659] Started watch for AppliedToGroup
2022-04-12T18:46:51.7005294Z I0412 18:46:36.180718    5020 networkpolicy_controller.go:646] Starting watch for NetworkPolicy
2022-04-12T18:46:51.7005857Z I0412 18:46:36.180718    5020 networkpolicy_controller.go:659] Started watch for NetworkPolicy
2022-04-12T18:46:51.7006686Z I0412 18:46:36.180718    5020 networkpolicy_controller.go:689] Received 1 init events for NetworkPolicy
2022-04-12T18:46:51.7007745Z I0412 18:46:36.180718    5020 networkpolicy_controller.go:244] NetworkPolicy K8sNetworkPolicy:ns1/policy1 applied to Pods on this Node
2022-04-12T18:46:51.7008423Z I0412 18:46:36.181718    5020 networkpolicy_controller.go:689] Received 1 init events for AppliedToGroup
2022-04-12T18:46:51.7009037Z I0412 18:46:36.181718    5020 networkpolicy_controller.go:689] Received 2 init events for AddressGroup
2022-04-12T18:46:51.7009699Z I0412 18:46:36.181718    5020 networkpolicy_controller.go:457] All watchers have completed full sync, installing flows for init events
2022-04-12T18:46:51.7010361Z I0412 18:46:36.181718    5020 networkpolicy_controller.go:461] Starting NetworkPolicy workers now
2022-04-12T18:46:51.7011000Z I0412 18:46:36.182718    5020 networkpolicy_controller.go:467] Starting IDAllocator worker to maintain the async rule cache
2022-04-12T18:46:51.7011641Z I0412 18:46:36.182718    5020 status_controller.go:209] Starting NetworkPolicy StatusController
2022-04-12T18:46:51.7012451Z I0412 18:46:36.183719    5020 status_controller.go:216] Shutting down NetworkPolicy StatusController
2022-04-12T18:46:51.7013033Z I0412 18:46:36.183719    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7013597Z I0412 18:46:36.187107    5020 networkpolicy_controller.go:429] Waiting for Antrea client to be ready
2022-04-12T18:46:51.7014143Z I0412 18:46:36.187107    5020 networkpolicy_controller.go:440] Antrea client is ready
2022-04-12T18:46:51.7014773Z I0412 18:46:36.187622    5020 networkpolicy_controller.go:455] Waiting for all watchers to complete full sync
2022-04-12T18:46:51.7016025Z I0412 18:46:36.187622    5020 networkpolicy_controller.go:646] Starting watch for AppliedToGroup
2022-04-12T18:46:51.7016591Z I0412 18:46:36.187622    5020 networkpolicy_controller.go:659] Started watch for AppliedToGroup
2022-04-12T18:46:51.7017119Z I0412 18:46:36.187622    5020 networkpolicy_controller.go:646] Starting watch for AddressGroup
2022-04-12T18:46:51.7017637Z I0412 18:46:36.187622    5020 networkpolicy_controller.go:659] Started watch for AddressGroup
2022-04-12T18:46:51.7018179Z I0412 18:46:36.187622    5020 networkpolicy_controller.go:689] Received 1 init events for AppliedToGroup
2022-04-12T18:46:51.7018715Z I0412 18:46:36.188619    5020 networkpolicy_controller.go:646] Starting watch for NetworkPolicy
2022-04-12T18:46:51.7019195Z I0412 18:46:36.188619    5020 networkpolicy_controller.go:659] Started watch for NetworkPolicy
2022-04-12T18:46:51.7019725Z I0412 18:46:36.188619    5020 networkpolicy_controller.go:689] Received 1 init events for NetworkPolicy
2022-04-12T18:46:51.7020331Z I0412 18:46:36.188619    5020 networkpolicy_controller.go:244] NetworkPolicy K8sNetworkPolicy:ns1/policy1 applied to Pods on this Node
2022-04-12T18:46:51.7020919Z I0412 18:46:36.188619    5020 networkpolicy_controller.go:689] Received 1 init events for AddressGroup
2022-04-12T18:46:51.7021506Z I0412 18:46:36.188619    5020 networkpolicy_controller.go:457] All watchers have completed full sync, installing flows for init events
2022-04-12T18:46:51.7022080Z I0412 18:46:36.188619    5020 networkpolicy_controller.go:461] Starting NetworkPolicy workers now
2022-04-12T18:46:51.7022648Z I0412 18:46:36.189619    5020 networkpolicy_controller.go:467] Starting IDAllocator worker to maintain the async rule cache
2022-04-12T18:46:51.7023159Z I0412 18:46:36.189619    5020 status_controller.go:209] Starting NetworkPolicy StatusController
2022-04-12T18:46:51.7023675Z E0412 18:46:36.205915    5020 networkpolicy_controller.go:726] Unknown event: {BOOKMARK <nil>}
2022-04-12T18:46:51.7024230Z I0412 18:46:36.205915    5020 networkpolicy_controller.go:663] Stopped watch for AddressGroup, total items received: 2
2022-04-12T18:46:51.7024770Z E0412 18:46:36.205915    5020 networkpolicy_controller.go:726] Unknown event: {BOOKMARK <nil>}
2022-04-12T18:46:51.7025393Z I0412 18:46:36.205915    5020 networkpolicy_controller.go:663] Stopped watch for AppliedToGroup, total items received: 2
2022-04-12T18:46:51.7026030Z I0412 18:46:36.205915    5020 networkpolicy_controller.go:195] NetworkPolicy K8sNetworkPolicy:ns1/policy2 applied to Pods on this Node
2022-04-12T18:46:51.7026699Z I0412 18:46:36.250060    5020 networkpolicy_controller.go:228] NetworkPolicy K8sNetworkPolicy:ns1/policy1 no longer applied to Pods on this Node
2022-04-12T18:46:51.7027376Z I0412 18:46:36.391799    5020 networkpolicy_controller.go:228] NetworkPolicy K8sNetworkPolicy:ns1/policy2 no longer applied to Pods on this Node
2022-04-12T18:46:51.7027938Z I0412 18:46:36.398460    5020 status_controller.go:216] Shutting down NetworkPolicy StatusController
2022-04-12T18:46:51.7028475Z I0412 18:46:47.749747    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7028965Z I0412 18:46:47.750751    5020 reconciler.go:840] Forgetting rule unknown-rule-id
2022-04-12T18:46:51.7029469Z I0412 18:46:47.750751    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7030005Z I0412 18:46:47.750751    5020 reconciler.go:840] Forgetting rule foo
2022-04-12T18:46:51.7030484Z I0412 18:46:47.751749    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7030944Z I0412 18:46:47.751749    5020 reconciler.go:840] Forgetting rule foo
2022-04-12T18:46:51.7031381Z I0412 18:46:47.752749    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7031832Z I0412 18:46:47.753749    5020 reconciler.go:840] Forgetting rule foo
2022-04-12T18:46:51.7032316Z I0412 18:46:47.754750    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7032954Z I0412 18:46:47.755749    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7033536Z I0412 18:46:47.755749    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7034117Z I0412 18:46:47.756749    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7034676Z I0412 18:46:47.756749    5020 reconciler.go:885] Can't find interface for ns1/pod2, skipping
2022-04-12T18:46:51.7035195Z I0412 18:46:47.757749    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7035774Z I0412 18:46:47.757749    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7036338Z I0412 18:46:47.758750    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7036923Z I0412 18:46:47.758750    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7037486Z I0412 18:46:47.759750    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7038059Z I0412 18:46:47.759750    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7038819Z W0412 18:46:47.759750    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:&PodReference{Name:pod1,Namespace:ns1,},ExternalEntity:nil,IPs:[],Ports:[]NamedPort{},Node:nil,}
2022-04-12T18:46:51.7039486Z I0412 18:46:47.760750    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7040007Z I0412 18:46:47.761750    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7040582Z I0412 18:46:47.762750    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7041152Z I0412 18:46:47.762750    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7041708Z I0412 18:46:47.763750    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7042278Z I0412 18:46:47.763750    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7042898Z I0412 18:46:47.764750    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7043466Z I0412 18:46:47.764750    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7043981Z I0412 18:46:47.765751    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7044540Z I0412 18:46:47.766751    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7045152Z I0412 18:46:47.767749    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7045721Z I0412 18:46:47.767749    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7046344Z I0412 18:46:47.768749    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7046905Z I0412 18:46:47.769751    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7047618Z W0412 18:46:47.769751    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 3]],Ports:[]NamedPort{},Node:nil,}
2022-04-12T18:46:51.7048303Z I0412 18:46:47.769751    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7049060Z W0412 18:46:47.769751    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 3]],Ports:[]NamedPort{},Node:nil,}
2022-04-12T18:46:51.7049786Z W0412 18:46:47.769751    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 3]],Ports:[]NamedPort{},Node:nil,}
2022-04-12T18:46:51.7050457Z I0412 18:46:47.770750    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7051164Z W0412 18:46:47.770750    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 3]],Ports:[]NamedPort{},Node:nil,}
2022-04-12T18:46:51.7051925Z W0412 18:46:47.770750    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 3]],Ports:[]NamedPort{},Node:nil,}
2022-04-12T18:46:51.7052555Z I0412 18:46:47.771750    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7053073Z I0412 18:46:47.773750    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7053643Z E0412 18:46:47.773750    5020 reconciler.go:364] rule ingress-rule already realized during the initialization phase
2022-04-12T18:46:51.7054199Z I0412 18:46:47.775751    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7054765Z I0412 18:46:47.775751    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7055401Z I0412 18:46:47.775751    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7055962Z I0412 18:46:47.776751    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7056540Z I0412 18:46:47.777751    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7057369Z I0412 18:46:47.777751    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7057971Z I0412 18:46:47.778751    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7058588Z I0412 18:46:47.778751    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7059254Z I0412 18:46:47.778751    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7059838Z I0412 18:46:47.778751    5020 reconciler.go:885] Can't find interface for ns1/pod4, skipping
2022-04-12T18:46:51.7060339Z I0412 18:46:47.779751    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7060961Z I0412 18:46:47.780751    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7061630Z I0412 18:46:47.780751    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7062763Z I0412 18:46:47.780751    5020 reconciler.go:908] Can't find interface for ns1/pod4, skipping
2022-04-12T18:46:51.7063318Z I0412 18:46:47.781751    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7064021Z I0412 18:46:47.781751    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7064686Z I0412 18:46:47.781751    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7065244Z I0412 18:46:47.782751    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7065920Z I0412 18:46:47.783752    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7066588Z I0412 18:46:47.783752    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7067192Z I0412 18:46:47.784752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7067851Z I0412 18:46:47.784752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy AntreaClusterNetworkPolicy:name1
2022-04-12T18:46:51.7068567Z I0412 18:46:47.784752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy AntreaClusterNetworkPolicy:name1
2022-04-12T18:46:51.7069733Z I0412 18:46:47.785752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7070418Z I0412 18:46:47.785752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy AntreaClusterNetworkPolicy:name1
2022-04-12T18:46:51.7071083Z I0412 18:46:47.786751    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy AntreaClusterNetworkPolicy:name1
2022-04-12T18:46:51.7071964Z W0412 18:46:47.786751    5020 reconciler.go:1063] Can not resolve port https for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 1]],Ports:[]NamedPort{NamedPort{Port:80,Name:http,Protocol:TCP,},},Node:nil,}
2022-04-12T18:46:51.7072988Z W0412 18:46:47.787751    5020 reconciler.go:1063] Can not resolve port https for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 2]],Ports:[]NamedPort{NamedPort{Port:80,Name:http,Protocol:TCP,},},Node:nil,}
2022-04-12T18:46:51.7073967Z W0412 18:46:47.787751    5020 reconciler.go:1063] Can not resolve port https for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 3]],Ports:[]NamedPort{NamedPort{Port:8080,Name:http,Protocol:TCP,},},Node:nil,}
2022-04-12T18:46:51.7074957Z W0412 18:46:47.787751    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 4]],Ports:[]NamedPort{NamedPort{Port:443,Name:https,Protocol:TCP,},},Node:nil,}
2022-04-12T18:46:51.7075845Z W0412 18:46:47.787751    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 5]],Ports:[]NamedPort{},Node:nil,}
2022-04-12T18:46:51.7076670Z W0412 18:46:47.787751    5020 reconciler.go:1063] Can not resolve port https for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 5]],Ports:[]NamedPort{},Node:nil,}
2022-04-12T18:46:51.7077564Z W0412 18:46:47.787751    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 6]],Ports:[]NamedPort{NamedPort{Port:443,Name:foo,Protocol:TCP,},},Node:nil,}
2022-04-12T18:46:51.7078525Z W0412 18:46:47.787751    5020 reconciler.go:1063] Can not resolve port https for endpoints &GroupMember{Pod:nil,ExternalEntity:nil,IPs:[[0 0 0 0 0 0 0 0 0 0 255 255 1 1 1 6]],Ports:[]NamedPort{NamedPort{Port:443,Name:foo,Protocol:TCP,},},Node:nil,}
2022-04-12T18:46:51.7079319Z I0412 18:46:47.789752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7080047Z I0412 18:46:47.789752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7080653Z I0412 18:46:47.790754    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7081302Z I0412 18:46:47.790754    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7081850Z I0412 18:46:47.790754    5020 reconciler.go:885] Can't find interface for ns1/pod2, skipping
2022-04-12T18:46:51.7082460Z I0412 18:46:47.791752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7083083Z I0412 18:46:47.791752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7083703Z I0412 18:46:47.793752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7084332Z I0412 18:46:47.793752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7084934Z I0412 18:46:47.794752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7085550Z I0412 18:46:47.794752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7086316Z W0412 18:46:47.794752    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:&PodReference{Name:pod1,Namespace:ns1,},ExternalEntity:nil,IPs:[],Ports:[]NamedPort{},Node:nil,}
2022-04-12T18:46:51.7087034Z I0412 18:46:47.795752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7087647Z I0412 18:46:47.795752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7088252Z I0412 18:46:47.796752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7088928Z I0412 18:46:47.797752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7089532Z I0412 18:46:47.798752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7090143Z I0412 18:46:47.798752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7090748Z I0412 18:46:47.799752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7091329Z I0412 18:46:47.799752    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7091937Z I0412 18:46:47.800753    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7092557Z I0412 18:46:47.800753    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7093165Z I0412 18:46:47.801753    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7093773Z I0412 18:46:47.802752    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7094375Z I0412 18:46:47.803752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7094994Z I0412 18:46:47.803752    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7095622Z I0412 18:46:47.803752    5020 reconciler.go:989] IPBlock 10.10.0.0/16 is using unsupported address family, skip it
2022-04-12T18:46:51.7096152Z I0412 18:46:47.805753    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7096763Z I0412 18:46:47.805753    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7097446Z I0412 18:46:47.806753    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7098079Z I0412 18:46:47.806753    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7098673Z I0412 18:46:47.806753    5020 reconciler.go:885] Can't find interface for ns1/pod2, skipping
2022-04-12T18:46:51.7099316Z I0412 18:46:47.807752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7100009Z I0412 18:46:47.807752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7100579Z I0412 18:46:47.809754    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7101204Z I0412 18:46:47.809754    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7101826Z I0412 18:46:47.810752    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7102456Z I0412 18:46:47.810752    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7103357Z W0412 18:46:47.810752    5020 reconciler.go:1063] Can not resolve port http for endpoints &GroupMember{Pod:&PodReference{Name:pod1,Namespace:ns1,},ExternalEntity:nil,IPs:[],Ports:[]NamedPort{},Node:nil,}
2022-04-12T18:46:51.7104028Z I0412 18:46:47.811754    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7104598Z I0412 18:46:47.811754    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7105337Z I0412 18:46:47.812753    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7105913Z I0412 18:46:47.813753    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7106529Z I0412 18:46:47.814754    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7107152Z I0412 18:46:47.814754    5020 reconciler.go:263] Reconciling rule ingress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7107769Z I0412 18:46:47.815755    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7108393Z I0412 18:46:47.815755    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7109065Z I0412 18:46:47.816756    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7109685Z I0412 18:46:47.816756    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7110297Z I0412 18:46:47.817767    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7110857Z I0412 18:46:47.818755    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7111453Z I0412 18:46:47.818755    5020 fqdn.go:176] "DNS server override provided by user" dnsServer="8.8.8.8:53"
2022-04-12T18:46:51.7112062Z I0412 18:46:47.819766    5020 reconciler.go:263] Reconciling rule egress-rule of NetworkPolicy K8sNetworkPolicy:ns1/name1
2022-04-12T18:46:51.7112681Z I0412 18:46:47.821754    5020 status_controller.go:209] Starting NetworkPolicy StatusController
2022-04-12T18:46:51.7113265Z I0412 18:46:48.323265    5020 status_controller.go:216] Shutting down NetworkPolicy StatusController
2022-04-12T18:46:51.7113851Z I0412 18:46:48.324269    5020 status_controller.go:209] Starting NetworkPolicy StatusController
2022-04-12T18:46:51.7114423Z I0412 18:46:48.824335    5020 status_controller.go:216] Shutting down NetworkPolicy StatusController
2022-04-12T18:46:51.7115235Z I0412 18:46:48.825338    5020 status_controller.go:209] Starting NetworkPolicy StatusController
2022-04-12T18:46:51.7115833Z I0412 18:46:49.326284    5020 status_controller.go:216] Shutting down NetworkPolicy StatusController
2022-04-12T18:46:51.7116418Z I0412 18:46:49.327287    5020 status_controller.go:209] Starting NetworkPolicy StatusController
2022-04-12T18:46:51.7116827Z FAIL
2022-04-12T18:46:51.7117269Z FAIL	antrea.io/antrea/pkg/agent/controller/networkpolicy	17.901s
```

--- a/pkg/agent/controller/networkpolicy/audit_logging_test.go
+++ b/pkg/agent/controller/networkpolicy/audit_logging_test.go
@@ -130,17 +130,6 @@ func newLogInfo(disposition string) (*logInfo, string) {
 	return testLogInfo, expected
 }
 
-func sendMultiplePackets(antreaLogger *AntreaPolicyLogger, ob *logInfo, numPackets int, sendInterval time.Duration) {
-	count := 0
-	for range time.Tick(sendInterval) {
-		count += 1
-		antreaLogger.LogDedupPacket(ob)
-		if count == numPackets {
-			break
-		}
-	}
-}
-
 func expectedLogWithCount(msg string, count int) string {
 	return fmt.Sprintf("%s [%d", msg, count)
 }
@@ -164,12 +153,17 @@ func TestDropPacketLog(t *testing.T) {
 }
 
 func TestDropPacketDedupLog(t *testing.T) {
-	antreaLogger, mockAnpLogger := newTestAntreaPolicyLogger(testBufferLength, &realClock{})
+	clock := NewVirtualClock(time.Now())
+	defer clock.Stop()
+	antreaLogger, mockAnpLogger := newTestAntreaPolicyLogger(testBufferLength, clock)
 	ob, expected := newLogInfo("Drop")
 	// Add the additional log info for duplicate packets.
 	expected = expectedLogWithCount(expected, 2)
 
-	go sendMultiplePackets(antreaLogger, ob, 2, time.Millisecond)
+	antreaLogger.LogDedupPacket(ob)
+	clock.Advance(time.Millisecond)
+	antreaLogger.LogDedupPacket(ob)
+	clock.Advance(testBufferLength)
 	actual := <-mockAnpLogger.logged
 	assert.Contains(t, actual, expected)
 }

